server/views/partials/navbar.jade

```diff
@@ -6,17 +6,62 @@ nav.navbar.navbar-default.navbar-fixed-top.nav-height
         a.navbar-brand(href='/challenges/next-challenge')
             img.img-responsive.nav-logo(src='https://s3.amazonaws.com/freecodecamp/freecodecamp_logo.svg', alt='learn to code javascript at freeCodeCamp logo')
     .collapse.navbar-collapse
-        ul.nav.navbar-nav.navbar-right.hamburger-dropdown
-            li.hidden-xs
-                a#nav-map-btn(href='/map' onclick='if (!(event.ctrlKey || event.metaKey)) {return false;}') Map
-            li.visible-xs
-                a(href='/map') Map
+        ul.nav.navbar-nav.navbar-right.hamburger-dropdown.show-on-small
+            li
+                a(href="https://forum.freecodecamp.com" target="_blank") Forum
+            li
+                a(href="https://guide.freecodecamp.org" target="_blank") Guide
+            li
+                a(href="https://medium.freecodecamp.com" target="_blank") Medium
             li
-                a(href='https://forum.freecodecamp.org', target='_blank') Forum
+                a(href="https://youtube.com/freecodecamp" target="_blank") YouTube
+            li
+                a(href="https://gitter.im/freecodecamp/home" target="_blank") Chat
+            li
+                a(href="https://forum.freecodecamp.com/t/free-code-camp-city-based-local-groups/19574" target="_blank") In your city
```

Comment:
Instead of this URL we should send them to: https://study-group-directory.freecodecamp.org/

Reply:
This is still not fixed. Please use this URL instead of that link to the forum: https://study-group-directory.freecodecamp.org/